test_e2e.sh

```diff
@@ -52,6 +52,14 @@ function assert_file_eq() {
     fail "file $1 does not contain '$2': $(cat $1)"
 }
 
+NCPORT=8888
+function freencport() {
```

Comment:
This is gross and racy and not guaranteed to work, but diminishes the risk to the point that I am not willing to argue about it any more.

Reply:
Thank you!